pkgs/tools/misc/turbo/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, fetchFromGitHub, buildGoModule, protobuf, protoc-gen-go, protoc-gen-go-grpc }:
+let
+  src = fetchFromGitHub {
+    owner = "vercel";
+    repo = "turbo";
+    rev = "69681a3396c377f5bc663bf3f942a62034694f29";
+    sha256 = "csapIeVB0FrLnmtUmLrRe8y54xmK50X30CV476DXEZI=";
+  };
+in
+buildGoModule {
+  pname = "turbo";
+  version = "1.6.3";
+
+  src = "${src}/cli";
```

Comment:
Can we not use something like `sourceRoot` here?

Reply:
Inlined fetchFromGitHub